packages/terra-consumer-nav/src/_variables.scss

```diff
@@ -0,0 +1,2 @@
+$nav-container-width: 320px;
+$mobile-breakpoint: 1024px;
```

Comment:
This should be a core variable we use... we'll want to sync it with whatever terra calls it.